website/src/pages/api/generate.ts

```diff
@@ -0,0 +1,31 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import * as path from "path";
+import * as fs from "fs";
+import * as mapshaper from "mapshaper";
+
+export default async function handler(
+  _req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const paths = {
+    shp: path.resolve("public/2020/g1l20.shp"),
+    dbf: path.resolve("public/2020/g1l20.dbf"),
+    prj: path.resolve("public/2020/g1l20.prj"),
+  };
```

Comment:
@herrstucki this works locally but not on Vercel. Not sure what I'm doing wrong here…